docs/getting-started/index.mdx

```diff
@@ -6,174 +6,377 @@ pagination_next: http/index
 import TabItem from "@theme/TabItem";
 import Tabs from "@theme/Tabs";
 
+import Link from "@docusaurus/Link";
+
 # Quickstart
 
-This quickstart will use the [ngrok agent](/agent/) to put your application on
-the internet and secure it so that only you can access it. It assumes you already have your
-own application running locally.
+<p className="text-lg">
+	This quickstart uses the [ngrok agent](/agent/) to put your application on the
+	internet with an agent endpoint, then secure it so that only you or others you
+	trust can access it. You'll need a few things to get started:
+</p>
 
-## Step 1: Install
+<ol className="marker:text-xl marker:font-bold marker:text-blue-500 pl-8">
```

Comment:
this layout looks a little strange to me. can we use something more simple like in https://ngrok.com/docs/traffic-policy/getting-started/#prerequisites ?
<img width="933" alt="image" src="https://github.com/user-attachments/assets/c7f502e6-04e2-40a9-aa08-a8f7b16bf3da">